.github/workflows/build_deploy.yml

```diff
@@ -6,7 +6,15 @@ on:
   workflow_dispatch:
     inputs:
       sa_enable:
+<<<<<<< HEAD
```

Comment:
solve merge conflict of this file please

Reply:
merge conflicts resolved